pkg/sink/cloudstorage/config.go

```diff
@@ -42,6 +42,12 @@ const (
 	minFlushInterval = 2 * time.Second
 	// the upper limit of flush-interval.
 	maxFlushInterval = 10 * time.Minute
+	// defaultFlushConcurrency is the default value of flush-concurrency.
+	defaultFlushConcurrency = 1
```

Comment:
what is the negative effect if the defalut value is bigger than one?

Reply:
The default behavior maintains compatibility with historical versions.